### ISSUE TITLE
TODO Docs

### ISSUE THREAD

**Issue by s-pace** [algolia/docsearch-scraper#359]
Hi,

I have written a small scripts in order to have every key currently using. We will need to detail all of them. Here is the outcome:
```js
{
   "index_name": "10xgenomics",
   "start_urls": [
      "https://support.10xgenomics.com/",
      {
         "url": "http://adonisjs.com/docs/(?P<version>.*?)/",
         "variables": {
            "version": [
               "3.0",
            ],
            [...]
         },
         "tags": [
         ],
         "selectors_key": "extensions",
         "page_rank": 1,
         "extra_attributes": {
            "documents": "learn",
            [...]
         }
      },
      [...]
   ],
   "stop_urls": [
      "/[0-9]\\.[0-9]/",
   ],
   "selectors": {
      "lvl0": {
         "selector": "",
         "default_value": "Support",
         "type": "xpath",
         "global": true,
         "attributes": {
            "crumb0": {
               "selector": ".//ol/li[3]/a",
               "type": "xpath"
            },
            [...]
         }
      },
      "lvl1": ".ans-body h1",
      "lvl2": ".ans-body h2",
      "lvl3": ".ans-body h3",
      "lvl4": ".ans-body h4",
      "lvl5": ".ans-body h5",
      "text": ".ans-body p, .ans-body li",
      "default": {
         "lvl0": ".content-inner h1",
         [...]
      },
      "method": {
         "lvl0": ".api-method-category",
         "lvl1": ".content-inner h1",
         "lvl2": ".rest-table-method .language-specific-content.php",
         "lvl3": ".content-inner h2",
         "lvl4": ".content-inner h3",
         "lvl5": ".content-inner h4",
         "lvl6": ".content-inner h5",
         "text": ".content-inner p, .content-inner li"
      },
      [...]
   "min_indexed_level": 2,
   "nb_hits": 4953,
   "selectors_exclude": [
      ".owner_badge",
    [...]
   ],
   "sitemap_urls": [
      "https://www.advancedinstaller.com/sitemap.xml"
   ],
   "js_render": true,
   "strip_chars": " .,;:#",
   "keep_tags": [
      "code"
   ],
   "custom_settings": {
      "attributeForDistinct": "hierarchy.lvl1",
      "ignorePlurals": false,
      "attributesToIndex": [
         "unordered(hierarchy_camel.lvl0)",
         "unordered(hierarchy.lvl0)",
         [...]
      ],
      "attributesToRetrieve": [
         "hierarchy",
         [...]
      ],
      "attributesForFaceting": [
         "language",
         [...]
      ],
      "customRanking": [
         "desc(weight.page_rank)",
         [...]
      ],
      "attributesToHighlight": [
         "hierarchy.lvl0",
         [...]
      ],
      "separatorsToIndex": "-",
      "ranking": [
         "words",
         [...]
      ],
      "attributesToSnippet": [
         "content:50"
      ],
      "synonyms": [
         [
            "caching",
            "caache"
         ],
         [...]
      ]
   },
   "use_anchors": true,
   "sitemap_urls_regexs": "/docs/",
   "js_wait": 1,
   "docker_memory": 2048,
   "min_indexed_lvl": 1,
   "css_exclude": [
      ".dxb-codesplit"
   ],
   "stop_content": [
      "Cannot find",
      [...]
   ],
   "conversation_id": [
      "465587457"
   ],
   "strict_redirects": true,
   "synonyms": [
      [
         "js",
         "javascript"
      ],
      [...]
   ],
   "docker_cpu": 2,
   "comments": [
      "The lvl0 will first try to select the active headings in the sidebar (Homepage, Core, Page, Code Pinball)",
      [...]
   ]
}

```

**Comment by s-pace** [comment]
I am going thought the closed issue from the 3 repo. Let's try to explain:

- Explain how the project is structured (3 repos)

- Other commands from ./docsearch

- Defining the required environment to launch the scrapper/crawler

- Feature regarding authentication

- docSearch-configs#244

- avoid using <code> markup (noise)

**Comment by s-pace** [comment]
Solved algolia/docsearch#269
